From 32f98288de65f7086dc63f7097bed954811320e7 Mon Sep 17 00:00:00 2001
Message-ID: <32f98288de65f7086dc63f7097bed954811320e7.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 00:45:38 +0100
Subject: [PATCH 12/19] gnu: go-github-com-cheekybits-is: Move to (gnu packages
 golang-check).

* gnu/packages/golang-check.scm (go-github-com-cheekybits-is): Move from
here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 33 +++++++++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 32 --------------------------------
 2 files changed, 33 insertions(+), 32 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 9fdb5b0d9e..d10eced626 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -1,4 +1,5 @@
 ;;; GNU Guix --- Functional package management for GNU
+;;; Copyright © 2018 Pierre Neidhardt <mail@ambrevar.xyz>
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
 ;;; Copyright © 2019 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2020 Jakub Kądziołka <kuba@kadziolka.net>
@@ -41,6 +42,38 @@ (define-module (gnu packages golang-check)
 ;;;
 ;;; Code:
 
+(define-public go-github-com-cheekybits-is
+  (let ((commit "68e9c0620927fb5427fda3708222d0edee89eae9")
+        (revision "0"))
+    (package
+      (name "go-github-com-cheekybits-is")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/cheekybits/is")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32
+           "1mkbyzhwq3rby832ikq00nxv3jnckxsm3949wkxd8ya9js2jmg4d"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/cheekybits/is"))
+      (home-page "https://github.com/cheekybits/is")
+      (synopsis "Mini testing helper for Go")
+      (description "A mini testing helper for Go.
+
+@itemize
+@item It has a simple interface (@command{is.OK} and @command{is.Equal}).
+@item It plugs into existing Go toolchain (uses @command{testing.T}).
+@item It's obvious for newcomers.
+@item It also gives you @command{is.Panic} and @command{is.PanicWith} helpers
+- because testing panics is ugly.
+@end itemize\n")
+      (license license:expat))))
+
 (define-public go-github-com-google-gofuzz
   (let ((commit "fd52762d25a41827db7ef64c43756fd4b9f7e382")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 1e78bfd5bd..d43b43f905 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -6806,38 +6806,6 @@ (define-public go-github-com-whyrusleeping-tar-utils
       (description "Tar utilities extracted from @command{go-ipfs} codebase.")
       (license license:expat))))
 
-(define-public go-github-com-cheekybits-is
-  (let ((commit "68e9c0620927fb5427fda3708222d0edee89eae9")
-        (revision "0"))
-    (package
-      (name "go-github-com-cheekybits-is")
-      (version (git-version "0.0.0" revision commit))
-      (source
-       (origin
-         (method git-fetch)
-         (uri (git-reference
-               (url "https://github.com/cheekybits/is")
-               (commit commit)))
-         (file-name (git-file-name name version))
-         (sha256
-          (base32
-           "1mkbyzhwq3rby832ikq00nxv3jnckxsm3949wkxd8ya9js2jmg4d"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/cheekybits/is"))
-      (home-page "https://github.com/cheekybits/is")
-      (synopsis "Mini testing helper for Go")
-      (description "A mini testing helper for Go.
-
-@itemize
-@item It has a simple interface (@command{is.OK} and @command{is.Equal}).
-@item It plugs into existing Go toolchain (uses @command{testing.T}).
-@item It's obvious for newcomers.
-@item It also gives you @command{is.Panic} and @command{is.PanicWith} helpers
-- because testing panics is ugly.
-@end itemize\n")
-      (license license:expat))))
-
 (define-public go-github-com-sabhiram-go-gitignore
   (let ((commit "525f6e181f062064d83887ed2530e3b1ba0bc95a")
         (revision "1"))
-- 
2.41.0

